portality/static/js/edges/public.article.edge.js

```diff
@@ -194,7 +194,9 @@ $.extend(true, doaj, {
                 edges.newResultsDisplay({
                     id: "results",
                     category: "results",
-                    renderer : doaj.renderers.newPublicSearchResultRenderer()
+                    renderer : doaj.renderers.newPublicSearchResultRenderer({
+                        "selector": selector
```

Comment:
See my comment above, this can be removed

Reply:
fixed